.github/dependabot.yml

```diff
@@ -39,9 +39,6 @@ updates:
         patterns:
           - '@testing-library/*'
           - '@types/testing-library*'
-          - '@types/jest'
```

Comment:
maybe handle dep related to testing like jsdom tsx ...

Reply:
Good catch! I'll fix that soon!